src/components/SideBar/SideBar.jsx

```diff
@@ -1,8 +1,16 @@
 import React from 'react'
+import { Link } from 'react-router-dom';
+
 
 const SideBar = () => {
+ 
+  
   return (
-    <div>SideBar</div>
+    <div className='sidebar-parent'>
+      <Link to={'/'}> <h3>Home</h3></Link>
+      <Link to={'/archive'}><h3>Archive</h3></Link>
+      <Link to={'/trash'}><h3>Trash</h3></Link>
```

Comment:
Since this is a normal string, you can write it as `to="/pathname"`, `{ }` are not required

Reply:
got it , changed.